lint/util.py

```diff
@@ -335,11 +335,12 @@ def tmpfile(cmd, code, filename, suffix='', output_stream=STREAM_STDOUT, env=Non
     if suffix:
         filename = os.path.splitext(filename)[0] + suffix
 
+    file = tempfile.NamedTemporaryFile(delete=False)
+    path = file.name
```

Comment:
After fixing stupid bugs in bash scripts today I am now fully confident I don't know what I am doing more ofter than I do, even if I think I do.